src/test/java/io/tarantool/driver/integration/ProxyTarantoolClientExampleIT.java

```diff
@@ -0,0 +1,109 @@
+            .withAddress(container.getHost(), container.getPort())
+            // For connecting to a Cartridge application, use the value of cluster_cookie parameter in the init.lua file
+            .withCredentials(container.getUsername(), container.getPassword())
+            // Specify using the default CRUD proxy operations mapping configuration
+            .withProxyMethodMapping()
+            // You may also specify more client settings, such as:
+            // timeouts, number of connections, custom MessagePack entities to Java objects mapping, etc.
+            .build();
+    }
+
+    private static void truncateSpace(String spaceName) {
+        client.space(spaceName).truncate().join();
+    }
+
+    @Test
+    public void clusterInsertSelectDeleteTest() throws ExecutionException, InterruptedException {
+        TarantoolSpaceOperations<TarantoolTuple, TarantoolResult<TarantoolTuple>> space = client.space(SPACE_NAME);
+
+        // Use TarantoolTupleFactory for instantiating new tuples
+        TarantoolTupleFactory tupleFactory = new DefaultTarantoolTupleFactory(
+            client.getConfig().getMessagePackMapper());
+
+        TarantoolTuple tuple;
+        List<TarantoolTuple> tarantoolTuples = new ArrayList<>();
+        for (int i = 0; i < 20; i++) {
+            // Pass the field corresponding to bucket_id as null for tarantool/crud to compute it automatically
+            tarantoolTuples.add(tupleFactory.create(1_000_000 + i, null, "FIO", 50 + i, 100 + i));
+        }
+        space.insertMany(tarantoolTuples).get();
+
+        // Primary index key value will be determined from the tuple
+        tuple = tupleFactory.create(1_000_000 - 1, null, "FIO", 50 - 1, 100 - 1);
+        Conditions conditions = Conditions.after(tuple);
+
+        TarantoolResult<TarantoolTuple> selectResult = space.select(conditions).get();
+        assertEquals(20, selectResult.size());
+
+        for (int i = 0; i < 20; i++) {
+            tuple = selectResult.get(i);
+            assertEquals(5, tuple.size());
+            assertEquals(1_000_000 + i, tuple.getInteger(0));
+            assertNotNull(tuple.getInteger(1)); //bucket_id
+            assertEquals("FIO", tuple.getString(2));
+            assertEquals(50 + i, tuple.getInteger(3));
+            assertEquals(100 + i, tuple.getInteger(4));
+        }
+
+        conditions = Conditions.greaterOrEquals(PK_FIELD_NAME, 1_000_000 + 10);
+        selectResult = space.select(conditions).get();
+        assertEquals(10, selectResult.size());
+
+        // space.delete can delete only a single tuple by primary index value
+        // Do not use conditions like after or greaterOrEquals
+        conditions = Conditions.equals(PK_FIELD_NAME, 1_000_000 + 15);
+        TarantoolResult<TarantoolTuple> deleteResult = space.delete(conditions).get();
+        assertEquals(1, deleteResult.size());
+
+        conditions = Conditions.indexEquals(PK_FIELD_NAME, Collections.singletonList(1_000_000 + 10));
```

Comment:
It should be work fine with this
```suggestion
        conditions = Conditions.indexEquals(PK_FIELD_NAME, 1_000_000 + 10);
```